client/lib/touch-detect/index.js

```diff
@@ -1,3 +1,5 @@
+/** @ssr-ready **/
```

Comment:
The changes here don't necessarily make it SSR-ready, does it? The condition below will still throw `ReferenceError` on the `window` reference. I think you want `'undefined' !== typeof window` instead?